Sources/AsyncQueue/ActorQueue.swift

```diff
@@ -80,65 +78,78 @@ public final class ActorQueue {
 
     // MARK: Public
 
+    /// Sets the actor context within which each `async` and `await`ed task will execute. Must be called from the target actor's `init` method.
+    /// - Parameter actor: The actor on which the queue's task will execute. This parameter is not retained by the receiver.
+    public func setTargetContext(to actor: ActorType) {
+        assert(target == nil) // Setting multiple targets on the same queue is API abuse.
+        target = actor
+    }
+
     /// Schedules an asynchronous task for execution and immediately returns.
     /// The scheduled task will not execute until all prior tasks have completed or suspended.
     /// - Parameter task: The task to enqueue.
-    public func async(_ task: @escaping @Sendable () async -> Void) {
-        taskStreamContinuation.yield(task)
+    public func async(_ task: @escaping @Sendable (isolated ActorType) async -> Void) {
+        taskStreamContinuation.yield(ActorTask(target: target!, task: task))
```

Comment:
Using force unwraps is fine here given the strict instructions for how an `ActorQueue` should be utilized:
* An `ActorQueue`'s lifecycle should not exceed the lifecycle of its `target`
* An `ActorQueue`'s `target` should be set in the `target` actor's `init`

Therefore, the `target` will always be available if the API is properly utilized.